circle.yml

```diff
@@ -39,3 +39,23 @@ deployment:
           -l "${CIRCLE_ARTIFACTS}"/acceptance_deploy 2>&1 >
           "${CIRCLE_ARTIFACTS}/acceptance_deploy.log"
       - build/push-aws.sh
+  osx:
+    branch: master
+    commands:
+      - |
+          if [ -n "${BUILD_OSX-}" ]; then
+            set -eux
+            # There must certainly be a better way to move ~/cockroach
```

Comment:
Actually, this is a valid point. I think we need to docker-in-docker to
make that work, but once we're inside the builder container, all the
directories are mounted correctly and no move is necessary.

On Sat, Feb 13, 2016 at 12:18 PM, Ben Darnell <notifications@github.com>
wrote:

> In circle.yml
> <https://github.com/cockroachdb/cockroach/pull/4343#discussion_r52830582>:
>
> > @@ -39,3 +39,23 @@ deployment:
> >            -l "${CIRCLE_ARTIFACTS}"/acceptance_deploy 2>&1 >
> >            "${CIRCLE_ARTIFACTS}/acceptance_deploy.log"
> >        - build/push-aws.sh
> > +  osx:
> > +    branch: master
> > +    commands:
> > +      - |
> > +          if [ -n "${BUILD_OSX-}" ]; then
> > +            set -eux
> > +            # There must certainly be a better way to move ~/cockroach
>
> Why do we have to move it at all? Why can't we reuse the source from the
> tests we've already run?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/4343/files#r52830582>.
>